tests/hcaptcha.spec.js

```diff
@@ -120,6 +126,27 @@ describe("hCaptcha", () => {
         expect(window.hcaptcha.remove.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
     });
 
+    it("can get Response", () => {
+        expect(window.hcaptcha.getResponse.mock.calls.length).toBe(0);
+        instance.getResponse();
+        expect(window.hcaptcha.getResponse.mock.calls.length).toBe(1);
+        expect(window.hcaptcha.getResponse.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
+    });
+
+    it("can get RespKey", () => {
+        expect(window.hcaptcha.getRespKey.mock.calls.length).toBe(0);
+        instance.getRespKey();
+        expect(window.hcaptcha.getRespKey.mock.calls.length).toBe(1);
+        expect(window.hcaptcha.getRespKey.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
+    });
+
+    it("can set Data", () => {
+        expect(window.hcaptcha.setData.mock.calls.length).toBe(0);
+        instance.setData();
```

Comment:
This test should also check that the object passed to `setData` is forwarded to the widget.
```js
const dataObj = { data: { nested: 1 } };
instance.setData(dataObj);
expect(window.hcaptcha.setData.mock.calls.length).toBe(1);
expect(window.hcaptcha.setData.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
expect(window.hcaptcha.setData.mock.calls[0][1]).toBe(dataObj);
```

Reply:
Done https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/09a2cab4b5e9dae2f5543b4e5839100ad0618c5a